### PR TITLE
[MVC-987] Impact page fixes

### PR DESCRIPTION
**Pull request by SydneyMhashu**
Fixes for page https://triggerise.org/impact/ 
Content is stashed to the left. Fixed the layout on web and mobile.

### CODE REVIEW

File: _sass/_blocks.scss [hunk 1]
```diff
@@ -1262,6 +1262,41 @@
...
+        font-size: map-get($font-scale, 10);
+      }
+    }
+
+}
+
+
+
+
+
```
Comment (raguiar9080) [inline _sass/_blocks.scss]: I've noticed you like to add a lot of newlines to separate some sections of the scss, maybe define a standard and use it throughout? I think 2 maybe 3 lines separation between any text should definitely separate things visually enough while still making things concise no?

**Review by raguiar9080 — APPROVED** [review]
Nice work on this Sydney 👍 
I left a comment regarding empty lines a bit throughout but it shouldn't affect functionality 😄 
On the PR itself, I would maybe add the screenshots present on the Jira ticket here as well to aid review 🙈 👍

File: _sass/_blocks.scss (hunk 1 above)
Comment (SydneyMhashu) [reply _sass/_blocks.scss]: Yes i agree. i usually add empty line so i know were my focus code is starting and ending. I am to remember to delete that after testing

**Comment by SydneyMhashu** [comment]
Screenshot 2021-01-22 at 15 24 43
Screenshot 2021-01-22 at 15 26 27

**Comment by SydneyMhashu** [comment]
Also included some screenshot for a general idea of the changes

**Review by xDavidRTx — APPROVED** [review]

**Pull request merged**